pkgs/top-level/all-packages.nix

```diff
@@ -42155,4 +42155,6 @@ with pkgs;
   insulator2 = callPackage ../applications/misc/insulator2 {};
 
   animdl = python3Packages.callPackage ../applications/video/animdl { };
+
+  tftui = callPackage ../by-name/tf/tftui/package.nix { };
```

Comment:
```suggestion
```

Reply:
Not sure what you're suggesting here.

Reply:
He's suggesting this to be removed because it's not needed when using the `by-name` hierarchy.

Reply:
Thanks removed it.